CMakeLists.txt

```diff
@@ -159,16 +119,9 @@ else()
     message(STATUS "Setting logging level to '${LOGGING_LEVEL}'")
 endif(NOT LOGGING_LEVEL)
 
-target_compile_definitions(rmm PUBLIC SPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_${LOGGING_LEVEL})
-
```

Comment:
This is more questionable, the question is who should prescribe the logging level -- `rmm` itself or the consumer of `rmm`. Probably the latter, so this should then go away here, but should, possibly, be added to the tests.

Reply:
OK, kept this down at the test/benchmarks level.  cuDF already has this.